docs/pages/versions/unversioned/guides/using-firebase.md

```diff
@@ -146,7 +146,7 @@ async function loginWithFacebook() {
     const credential = firebase.auth.FacebookAuthProvider.credential(token);
 
     // Sign in with credential from the Facebook user.
-    firebase.auth().signInWithCredential(credential).catch((error) => {
+    firebase.auth().signInAndRetrieveDataWithCredential(credential).catch((error) => {
```

Comment:
In the next version of Firebase they deprecate this method in favor of the old method `signInWithCredential`  🙃

Reply:
It's good to know. I'll close this PR, thanks @EvanBacon !